src/IECoreHoudini/LiveScene.cpp

```diff
@@ -463,7 +463,7 @@ bool LiveScene::hasTag( const Name &name, int filter ) const
 							continue;
 						}
 						
-						const UT_String &groupName = group->getName();
+						const UT_String groupName = group->getName().c_str();
```

Comment:
This is because group->getName() returns a UT_StringHolder, which doesn't have the substr function.